apps/openmw/mwmechanics/aiwander.hpp

```diff
@@ -75,6 +75,21 @@ namespace MWMechanics
             mState = wanderState;
             mIsWanderingManually = isManualWander;
         }
+
+        AiWanderStorage(const AiWanderStorage& storage)
+        {
+            mReaction = storage.mReaction;
+            mState = storage.mState;
+            mIsWanderingManually = storage.mIsWanderingManually;
+            mCanWanderAlongPathGrid = storage.mCanWanderAlongPathGrid;
+            mIdleAnimation = storage.mIdleAnimation;
+            mBadIdles = storage.mBadIdles;
+            mPopulateAvailableNodes = mPopulateAvailableNodes;
+            mAllowedNodes = storage.mAllowedNodes;
+            mTrimCurrentNode = mTrimCurrentNode;
+            mDoorCheckDuration = mDoorCheckDuration;
+            mStuckCount = mStuckCount;
```

Comment:
Is it a typo or redundant code? Assign object to itself makes no difference here.